src/components/common/Logo.tsx

```diff
@@ -0,0 +1,19 @@
+import Image from 'next/image';
+
+import LogoImage from '/public/images/fresh-beginnings-logo.png';
+
+const Logo = () => {
+  return (
+    <div className="relative bottom-36 lg:left-96 md:left-0">
+      <Image
```

Comment:
Logo looks great! Can we adjust a few things? There is a typo, should be "Toxin free" not "Toxic", can we adjust the border radius so it's not curved (nothing in our design is curved so we want to keep that standard), and not sure if this is using our default font, which should be `font-family: ui-serif, Georgia, Cambria, "Times New Roman", Times, serif;`